GUI/MetaDraw/PsmAnnotationViewModel.cs

```diff
@@ -293,7 +293,11 @@ public void DrawPeptideSpectralMatchPdf(MsDataScan msDataScan, PsmFromTsv psm, s
         {
             OxyColor ionColor;
 
-            if (productTypeDrawColors.ContainsKey(matchedIon.NeutralTheoreticalProduct.ProductType))
+            if(psmToDraw.VariantCrossingIons.ContainsKey(matchedIon) && psmToDraw.VariantCrossingIons[matchedIon])
```

Comment:
` && psmToDraw.VariantCrossingIons[matchedIon]` doesn't do anything

Reply:
same response as the comment above